mantidimaging/core/io/saver.py

```diff
@@ -198,7 +198,7 @@ def make_dirs_if_needed(dirname=None, overwrite_all=False):
         os.makedirs(path)
     elif os.listdir(path) and not overwrite_all:
         raise RuntimeError("The output directory is NOT empty:{0}\n. This can be "
```

Comment:
```suggestion
        raise RuntimeError("The output directory is NOT empty:{0}\nThis can be "
```
Just thought the full stop on the new line looked a bit weird. Everything else looks good to me.